### DISCUSSION TITLE
is Supported ?

### DISCUSSION THREAD

**Discussion by Deepankar1993** [Live-Charts/LiveCharts2#216] [General]
Live chart v2 is supported on .netcore or its only on .netframework?

**Comment by CodeWithMichal** [comment]
Yes, it is.
Here you can find nuget packages for currently available platforms : https://github.com/beto-rodriguez/LiveCharts2/issues/35

You can check there minimum necessary Framework/Core version

**Reply by Deepankar1993** [reply]
thanks
